x-pack/docs/en/watcher/trigger/schedule.asciidoc

```diff
@@ -15,7 +15,7 @@ that's more frequent than the throttle period, the throttle period overrides the
 schedule. For example, if you set the throttle period to one minute (60000 ms)
 and set the schedule to every 10 seconds, the watch is executed no more than
 once per minute. For more information about throttling, see
-<<actions-ack-throttle, Acknowledgement and Throttling>>.
+<<actions-ack-throttle,Acknowledgement and throttling>>.
```

Comment:
We can probably just use the default link text here. It's a bit more future-proof.
```suggestion
<<actions-ack-throttle>>.
```